index.html

```diff
@@ -12933,7 +12933,7 @@ <h2 id="Performance">
           When an <a href="#acquire-the-content">acquire the content</a>
           operation is performed on an <a>AudioBuffer</a>, the entire operation
           can usually be implemented without copying channel data. In
-          particular, the last step should be performed lazily at the next
+          particular, the last step SHOULD be performed lazily at the next
```

Comment:
seems like a SHOULD to me since other UA strategies could conceivably deliver the desired goal of minimizing copies and allocations... no normative difference could be observed if the UA failed to do so